new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -2211,7 +2238,7 @@ public static void createDomainJob(String image, boolean isUseSecret, String pvN
                     .containers(Arrays.asList(jobContainer  // container containing WLST or WDT details
                         .name("create-weblogic-domain-onpv-container")
                         .image(image)
-                        .imagePullPolicy("Always")
+                        .imagePullPolicy("IfNotPresent")
                         .ports(Arrays.asList(new V1ContainerPort()
```

Comment:
would it work on multinoded cluster? why switch from Always?

Reply:
fixed